packages/ui/cart/drawer/CartLineItem.tsx

```diff
@@ -99,7 +99,8 @@ export function CartLineItem({ lineItem }: CartLineItemProps) {
                            fontSize="sm"
                            borderRadius="sm"
                         >
-                           {lineItem.name}
+                           {lineItem.name}{' '}
+                           {lineItem.variant && `| ${lineItem.variant}`}
```

Comment:
This looks more correct to me since it won't always force the space.
```suggestion
                           {lineItem.name}
                           {lineItem.variant && ` | ${lineItem.variant}`}
```